.github/workflows/pycodestyle_workflow.yml

```diff
@@ -19,7 +19,9 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
     - name: Install Poetry
-      uses: snok/install-poetry@v1.3.0
+      uses: snok/install-poetry@v1
+      with:
+        version: 1.8.3
```

Comment:
This was because of a bug in later versions of Poetry? Worth puting a comment about this pin so we remember?